src/main/java/slimeknights/tconstruct/tools/data/Materials.java

```diff
@@ -16,61 +16,54 @@
 final class Materials {
   static final List<IMaterial> allMaterials = new ArrayList<>();
 
-  // natural resources/blocks
+  //Tier 1
   public static final IMaterial wood = mat(MaterialIds.wood, true, "8e661b");
-  public static final IMaterial stone = mat(MaterialIds.stone, true, "999999");
-  public static final IMaterial flint = mat(MaterialIds.flint, true, "696969");
-  public static final IMaterial cactus = mat(MaterialIds.cactus, true, "00a10f");
   public static final IMaterial bone = mat(MaterialIds.bone, true, "ede6bf");
-  public static final IMaterial obsidian = mat(MaterialIds.obsidian, TinkerFluids.moltenObsidian, true, "601cc4");
-  public static final IMaterial prismarine = mat(MaterialIds.prismarine, true, "7edebc");
-  public static final IMaterial endstone = mat(MaterialIds.endstone, true, "e0d890");
+  public static final IMaterial stone = mat(MaterialIds.stone, true, "999999");
   public static final IMaterial paper = mat(MaterialIds.paper, true, "ffffff");
-  public static final IMaterial sponge = mat(MaterialIds.sponge, true, "cacc4e");
-  public static final IMaterial firewood = mat(MaterialIds.firewood, true, "cc5300");
-
-  // Slime
-  public static final IMaterial knightslime = mat(MaterialIds.knightslime, TinkerFluids.moltenKnightslime, false, "f18ff0");
-  public static final IMaterial slime = mat(MaterialIds.slime, true, "82c873");
-  public static final IMaterial blueslime = mat(MaterialIds.blueslime, true, "74c8c7");
-  public static final IMaterial magmaslime = mat(MaterialIds.magmaslime, true, "ff960d");
+  public static final IMaterial flint = mat(MaterialIds.flint, true, "696969");
+  public static final IMaterial coral = mat(MaterialIds.coral, true, "ffef38");
 
-  // Metals
-  public static final IMaterial iron = mat(MaterialIds.iron, TinkerFluids.moltenIron, false, "cacaca");
-  public static final IMaterial copper = mat(MaterialIds.copper, TinkerFluids.moltenCopper, false, "ed9f07");
-  public static final IMaterial pigiron = mat(MaterialIds.pigiron, TinkerFluids.moltenPigIron, false, "ef9e9b");
+  //Tier 2
+  public static final IMaterial copper = mat(MaterialIds.copper, false, "ed9f07");
```

Comment:
Missing the fluid, this will make it uncastable. Same in many other cases in this file.